utils/index.ts

```
@@ -62,25 +63,30 @@ export const OpenAIStream = async (
+    body['stream'] = true;
+  }
+
+  if (model === 'grok-3-mini-beta') {
+    body['reasoning_effort'] = 'high';
+    body['temperature'] = 0.7;
+  }
 
-const apiUrl = model === 'grok-2-latest' ? 'https://api.x.ai/v1/chat/completions' : model === 'deepseek-chat' ? 'https://api.deepseek.com/chat/completions' : 'https://api.openai.com/v1/chat/completions';
-const apiKey = model === 'grok-2-latest' ? process.env.X_AI_API_KEY : model === 'deepseek-chat' ? process.env.DEEPSEEK_API_KEY : key || process.env.OPENAI_API_KEY;
+  const apiUrl = model === 'grok-2-latest' || 'grok-3-mini-beta' ? 'https://api.x.ai/v1/chat/completions' : model === 'deepseek-chat' ? 'https://api.deepseek.com/chat/completions' : 'https://api.openai.com/v1/chat/completions';
+  const apiKey = model === 'grok-2-latest' || 'grok-3-mini-beta' ? process.env.X_AI_API_KEY : model === 'deepseek-chat' ? process.env.DEEPSEEK_API_KEY : key || process.env.OPENAI_API_KEY;
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)